src/config.ts

```diff
@@ -0,0 +1,13 @@
+export interface Config {
```

Comment:
instead of tying this to env vars - we decided to update the ActionsAPI `constructor` to take an additional `config` argument shaped like this, and then update `action-server` to pass it in when constructing the object